pkg/reconciler/certificate/config/zz_generated.deepcopy.go

```diff
@@ -2,7 +2,7 @@
 // +build !ignore_autogenerated
 
 /*
-Copyright 2020 The Knative Authors
+Copyright 2022 The Knative Authors
```

Comment:
Could you pls bump the year in boilerplate template? https://github.com/knative/serving/blob/main/hack/boilerplate/boilerplate.go.txt#L2

Reply:
Sure

Reply:
I guess we need to do this for boilerplate.sh.txt too. Should we do it in a separate PR?

Reply:
Sure, that's a good suggestion. I haven't realized the obvious regeneration impact. :)